src/app/mat-main-nav/desktop-menu/desktop-menu.component.scss

```diff
@@ -0,0 +1,23 @@
+@import "@angular/material/core/style/_variables";
+@import "~nhsuk-frontend/packages/core/settings/colours";
```

Comment:
this import doesn't seem to be used in this file

Reply:
done